scalameta/tokens/shared/src/main/scala/scala/meta/tokens/Tokens.scala

```diff
@@ -29,7 +29,11 @@ import scala.meta.internal.prettyprinters._
 extends immutable.IndexedSeq[Token] with collection.IndexedSeqOptimized[Token, immutable.IndexedSeq[Token]] {
   def apply(idx: Int): Token = tokens(start + idx)
   def length: Int = end - start
-  override def slice(from: Int, until: Int): Tokens = Tokens(tokens, start + math.max(from, 0), start + math.min(math.max(until, 0), length))
+  override def slice(from: Int, until: Int): Tokens = {
+    val lo = start + ((from max 0) min length)
```

Comment:
Nitpick, can we use non-infix here?

```scala
    val lo = start + from.max(0).min(length)
    val hi = lo.max(start + until.min(length))
```

Infix should be reserved for symbolic operators.

Reply:
done